device.go

```diff
@@ -79,6 +79,17 @@ type devIndexEntry struct {
 }
 type devIndex map[string]devIndexEntry
 
+// Guest-side PCI path, identifies a PCI device by where it sits in
+// the PCI topology.
+//
+// Has the format "bridgeAddr/deviceAddr" where bridgeAddr is the
```

Comment:
I thought from #854 that you could have multiple bridge addresses (device accessed with more than one bridge). So I think you see that as just a current limitation of the implementation. In that case, simply state "Currently has the format..." and then explain where this is going?

Reply:
Well, I figure that will become obvious when I add the patches to actually do that.  It's `do-not-merge` for a reason..